src/bundle/ui-dev/src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -192,16 +192,23 @@ const UniversalDiscoveryModule = (props) => {
     };
 
     useEffect(() => {
+        const setContentTypesInfo = (contentTypes) => {
```

Comment:
maybe to be less confusing 🤔  
```suggestion
        const addContentTypesInfo = (contentTypes) => {
```